docs/docs-content/clusters/public-cloud/aws/create-cluster.md

```diff
@@ -100,7 +100,7 @@ Use the following steps to provision a new AWS cluster:
   
 
 
-10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool.
+10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools. 
```

Comment:
```suggestion
10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in both the master pool and across all worker pools. 
```

Reply:
To clarify, "...in both" - Does this mean the master pool needs 4 CPUs and 4 GB of memory and the work pool needs the same  - 4 CPUs and 4 GB of memory - for each node?

Reply:
The master pool needs for 4CPUs and 4GB across all nodes, and the worker pool needs 4CPUs and 4GB of memory across all nodes across all pools.